pandas/core/computation/eval.py

```diff
@@ -148,7 +147,7 @@ def _check_for_locals(expr, stack_level, parser):
 
 def eval(expr, parser='pandas', engine=None, truediv=True,
          local_dict=None, global_dict=None, resolvers=(), level=0,
-         target=None, inplace=None):
+         target=None, inplace=False):
```

Comment:
As I said before, I don't think we need to check the modifyability of `target`. Leave it to the user.